features/graph/common/src/main/java/org/opennms/oce/features/graph/common/GraphMLConverter.java

```diff
@@ -66,6 +65,8 @@
     private final Map<ResourceKey, GraphMLNode> inventoryObjectsForAlarms = new HashMap<>();
     private final SetMultimap<GraphMLNode, GraphMLNode> garbageCollectedAlarmInventoryEdges = MultimapBuilder.hashKeys().hashSetValues().build(); 
 
+    private final EdgeTimeNormalizer edgeTimeNormalizer;
```

Comment:
Can we rename this variable so there's no confusion between the `edgeTimeNormalizer` defined in `handleSituation` and the one defined in `createAlarmEdge`?